src/test/java/ua/lviv/iot/parkingServer/grpcservice/VehicleGrpcServiceTest.kt

```diff
@@ -0,0 +1,121 @@
+        stub = VehicleGrpcServiceGrpc.newBlockingStub(channel)
+    }
+
+    @Test
+    fun createVehicle() {
+        val vehicle = Vehicle(
+            number = "KA6706VN",
+            typeOfVehicle = VehicleType.BUS,
+            durationOfUseOfParkingSpot = Duration.ofSeconds(3600),
+            isTicketReceived = true
+        )
+
+        val vehicleRequest = VehicleOuterClass.VehicleRequest.newBuilder()
+            .setVehicle(vehicleConverter.vehicleToProto(vehicle))
+            .build()
+
+        val response = stub.createVehicle(vehicleRequest)
+
+        assertEquals("KA6706VN", response.vehicle.number)
+        assertEquals(VehicleOuterClass.VehicleType.BUS, response.vehicle.typeOfVehicle)
+        assertEquals(3600, response.vehicle.durationOfUseOfParkingSpot.seconds)
+        assertEquals(true, response.vehicle.isTicketReceived)
+    }
+
+    @Test
+    fun getVehicleById() {
+        val vehicle = Vehicle(
+            number = "KA6706VN",
+            typeOfVehicle = VehicleType.BUS,
+            durationOfUseOfParkingSpot = Duration.ofSeconds(3600),
+            isTicketReceived = true
+        )
+        vehicleRepository.save(vehicle).block()
+        val request = VehicleOuterClass.GetByIdVehicleRequest.newBuilder()
+            .setVehicleId(vehicle.id)
+            .build()
+        val response = stub.getVehicleById(request)
+        assertEquals("KA6706VN", response.vehicle.number)
+        vehicleRepository.deleteById(vehicle.id).block()
+    }
+
+    @Test
+    fun updateVehicle() {
+        val updatedVehicle = Vehicle(
+            number = "KA6706VW",
+            typeOfVehicle = VehicleType.BUS,
+            durationOfUseOfParkingSpot = Duration.ofSeconds(3600),
+            isTicketReceived = true
+        )
+        vehicleRepository.save(updatedVehicle).block()
+        val request = VehicleOuterClass.UpdateVehicleRequest.newBuilder()
+            .setVehicleId(updatedVehicle.id)
+            .setVehicle(vehicleConverter.vehicleToProto(updatedVehicle))
+            .build()
+
+        val response = stub.updateVehicle(request)
+        assertEquals("KA6706VW", response.vehicle.number)
+        vehicleRepository.deleteById(updatedVehicle.id).block()
+    }
+
+    @Test
+    fun deleteVehicle() {
+        val vehicle = Vehicle(
+            number = "KA6706VW",
+            typeOfVehicle = VehicleType.BUS,
+            durationOfUseOfParkingSpot = Duration.ofSeconds(3600),
+            isTicketReceived = true
+        )
+        vehicleRepository.save(vehicle).block()
+        val sizeBeforeDeletion = vehicleRepository.findAll().collectList().block()!!.size
+        val request = VehicleOuterClass.DeleteVehicleRequest.newBuilder()
+            .setVehicleId(vehicle.id)
+            .build()
+
+        stub.deleteVehicle(request)
+        val sizeAfterDeletion = vehicleRepository.findAll().collectList().block()!!.size
+        Assertions.assertThat(sizeBeforeDeletion).isGreaterThan(sizeAfterDeletion)
```

Comment:
in some test methods you use 
`org.junit.jupiter.api.Assertions.assertEquals`
in some another 
`org.assertj.core.api.Assertions`.

It's better to choose something one :)

Reply:
✅